tests/integration/cli.py

```diff
@@ -0,0 +1,24 @@
+import ast
+import pytest
+import subprocess
+
+PROJECT1 = 'foo'
+
+
+class TestCLI:
+    def test_add_list_delete_projects(self):
+        output = subprocess.check_output(['haas', 'list_projects'])
+        projects = ast.literal_eval(output)
+        size = len(projects)
+
+        subprocess.check_call(['haas', 'project_create', PROJECT1])
+        output = subprocess.check_output(['haas', 'list_projects'])
+        projects = ast.literal_eval(output)
+        assert PROJECT1 in projects
+        assert len(projects) == size + 1
```

Comment:
I was trying to make no assumptions on the state of the system before the test is run.